business/workloads.go

```diff
@@ -145,6 +158,9 @@ func (in *WorkloadService) GetWorkloadList(criteria WorkloadCriteria) (models.Wo
 		}
 		workloadList.Workloads = append(workloadList.Workloads, *wItem)
 	}
+	authpolicies = istioConfigList.AuthorizationPolicies
+	validations := in.getWorkloadValidations(authpolicies, *workloadList, criteria.Namespace)
```

Comment:
@shireenf-ibm as this feature will be optional, and by default this validation will be skipped, we need to add here a call to a function called "StripIgnoredChecks".

For reference, this is done in "runObjectCheckers" at istio_validations.go